awx/ui_next/src/screens/Inventory/InventoryHostAdd/InventoryHostAdd.jsx

```diff
@@ -1,8 +1,42 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
 import { CardBody } from '@patternfly/react-core';
+import InventoryHostForm from '../shared/InventoryHostForm';
+import { Config } from '@contexts/Config';
+import { InventoriesAPI } from '@api';
 
 function InventoryHostAdd() {
-  return <CardBody>Coming soon :)</CardBody>;
+  const [formError, setFormError] = useState(null);
+  const history = useHistory();
```

Comment:
I wanted to try out a new method of retrieving history and params by using [react-router hooks](https://reacttraining.com/react-router/web/api/Hooks). 

Usually, we would get access to the params and history object by retrieving props passed down from `withRouter` HOC. With hooks, we no longer need to wrap components with `withRouter`. 
* `useParams` hook gives us access to the `params` of the closest Route
* `useHistory` hook gives us access to the `history` object.